package.json

```diff
@@ -47,13 +47,15 @@
     ]
   },
   "devDependencies": {
+    "cypress": "^8.4.1",
     "eslint": "^7.32.0",
     "eslint-config-standard": "^16.0.3",
     "eslint-plugin-import": "^2.24.2",
     "eslint-plugin-json": "^3.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.25.1",
-    "prettier": "2.4.0"
+    "prettier": "2.4.0",
+    "typescript": "^4.4.3"
```

Comment:
Is typescript needed? I don't see it being used?

Reply:
It looks like it is needed for the cypress UI to run. I tried to remove it but the Cypress UI was not functional.